packages/components-css/paragraph-css/src/_mixin.scss

```diff
@@ -22,6 +22,14 @@
   line-height: var(--nl-paragraph-small-line-height, var(--nl-paragraph-line-height, inherit));
 }
 
-@mixin nl-paragraph__b {
+/*
+ * In SCSS the following mixins will overwrite each other:
+ *
+ * 1. `@mixin nl-paragraph--lead`
+ * 2. `@mixin nl-paragraph--lead`
```

Comment:
Een van deze twee moet `__` zijn :] maar point still stands. Dit gaat veel vaker voorkomen lijkt me en is `-scss-workaround` dan niet heel erg verbose en lelijk?

Reply:
ik sta open voor suggesties

Reply:
en ik denk: niet heel vaak, ik kan me zo geen ander voorbeeld bedenken